code/modules/unit_tests/stomach.dm

```diff
@@ -1,13 +1,13 @@
 /datum/unit_test/stomach/Run()
 	var/mob/living/carbon/human/human = allocate(/mob/living/carbon/human)
-	var/obj/item/reagent_containers/food/snacks/hotdog/fooditem = allocate(/obj/item/reagent_containers/food/snacks/hotdog)
+	var/obj/item/food/hotdog/fooditem = allocate(/obj/item/food/hotdog)
 	var/obj/item/organ/stomach/belly = human.getorganslot(ORGAN_SLOT_STOMACH)
 	var/obj/item/reagent_containers/pill/pill = allocate(/obj/item/reagent_containers/pill)
 	var/datum/reagent/drug/methamphetamine/meth = /datum/reagent/drug/methamphetamine
 
 	TEST_ASSERT_EQUAL(human.has_reagent(/datum/reagent/consumable/ketchup), FALSE, "Human somehow has ketchup before eating")
 
-	fooditem.attack(human, human)
+	SEND_SIGNAL(fooditem, COMSIG_ATOM_ATTACK_HAND)
```

Comment:
Note to Floyd who is adding the debug dog and I believe fixing this test: change this back to attack.

Reply:
ok, i was wondering why it was a signal